### DISCUSSION TITLE
Anyone's project disappear from cloud.walletconnect.com?

### DISCUSSION THREAD

**Discussion by jordangrant** [WalletConnect/walletconnect-monorepo#3202] [Developer Support] [answered]
<img width="1145" alt="Screenshot 2023-07-18 at 3 56 49 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/7986493/51503f4f-c694-4382-b7d9-e2a58de6a859">

**Comment by jordangrant** [comment]
<img width="1087" alt="Screenshot 2023-07-18 at 3 59 44 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/7986493/e1b4db10-9f8f-4fd0-8f99-d4876eb542d9">

@ HQ please remove this buggy modal for now

**Comment by jordangrant** [comment]
Seemingly fixed in record time thanks team

**Comment by Cali93** [comment, marked as answer]
Sorry for the inconvenience. There was a bug in our latest release. The issue has been fixed.
